.scalafmt.conf

```diff
@@ -1,4 +1,4 @@
-version = 2.1.0
```

Comment:
I'm in favor of keeping this config in sync across multiple repos of the Akka organization. For example, copy/paste the conf from https://github.com/akka/akka-grpc/blob/master/.scalafmt.conf

(I just realized akka/akka is still stuck in `2.1.0` so we can't use that as a baseline)

Reply:
I wonder if there is some new scalafmt configuration we're missing to avoid too much churn in the codebase.  It's quite a jump in versions so I expect some rules have changed in various ways.  For example, to not push parens to the next line.